src/components/canrawview/canrawview.h

```diff
@@ -18,9 +20,13 @@ class CanRawView : public QWidget {
 public Q_SLOTS:
     void frameReceived(const QCanBusFrame& frame);
     void frameSent(bool status, const QCanBusFrame& frame, const QVariant& context);
+    void simulationStarted(void);
```

Comment:
i thought this names would be good, becouse object of this class does not starts the simulation itself, it waits for simulation to get started.